addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -83,17 +83,17 @@
 		<item-type>Number</item-type>
 		<label>Total Usage</label>
 		<description>Used energy in Watt hours</description>
-		<state pattern="%d Wh" readOnly="true"></state>
+		<state pattern="%f Wh" readOnly="true"></state>
 	</channel-type>
 
 	<channel-type id="instantamp" advanced="true">
 		<item-type>Number</item-type>
 		<label>Instant Amp</label>
 		<description>Instant current in Amperes</description>
-		<state pattern="%d A" readOnly="true"></state>
+		<state pattern="%f A" readOnly="true"></state>
 	</channel-type>
 
-	<channel-type id="totalamphours" advanced="true">
```

Comment:
Is there a reason for de-pluralising this channel? It doesn't seem to make any functional difference, and it seems more natural to me to say amphours than amphour. Same comment applies to all the de-pluralisations, I'd be happier if the pull request changed instantamp to instantamps instead, as that is currently inconsistent throughout the plugin.